src/app/routes/onDemandAudio/getInitialData/index.js

```diff
@@ -134,6 +134,8 @@ export default async ({ path: pathname, pageType, service, toggles }) => {
           ? getPodcastPageIdentifier(pageIdentifier)
           : pageIdentifier,
         imageUrl: get(['content', 'blocks', 0, 'imageUrl'], LOG_LEVELS.INFO),
+        // An empty string in alt text will not be read out by AT; a null value will fall back to the default
```

Comment:
How about "An empty string/null value for alt text results in the image not being acknowledged by AT;"

What is the default referring to @ryanmccombe?

Reply:
@greenc05 we have default service level alt text for all images on Simorgh. So when the value is null, that default will be used. But when the value is an empty string i.e. `''` then the default alt text that simorgh uses will not be applied. And that's what the rest of this PR ensures.

E.g. The default alt image for bbc.com/afrique is 'BBC News Afrique'. If an image on simorgh doesn't come with alt text (i.e. `''` or `null`), we usually apply this to it. This is true for the podcast and on demand radio image as well. This PR differentiates between `null` and `''` by using the empty string `''` when it's given and using 'BBC News Afrique' when it's null. That way we can change the behaviour for podcast images without changing the behaviour for the rest of the site

Reply:
Thanks for the detailed explanation @pharingee